pkgs/top-level/ocaml-packages.nix

```diff
@@ -910,6 +910,8 @@ let
 
     js_of_ocaml-compiler = callPackage ../development/tools/ocaml/js_of_ocaml/compiler.nix {};
 
+    wasm_of_ocaml-compiler = callPackage ../development/tools/ocaml/js_of_ocaml/compiler-wasm.nix {};
```

Comment:
The entries in this file are sorted alphabetically: this one might be moved a bit further down.